docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -0,0 +1,155 @@
+
+
+#### 4. Required runtime encryption secret
+
+Model in Image requires a runtime encryption secret with a secure `password` key. This secret is used by the operator to encrypt model and domain home artifacts before it adds them to a runtime config map or log. The `password` that you set can safely change any time after you've fully shut down a domain but must remain the same for the life of a running domain. The runtime encryption secret that you create can be named anything but note that it is a best practice to name and label secrets with their domain UID to help ensure that cleanup scripts can find and delete them.
+
+**NOTE**: Because the runtime encryption password does not need to be shared and only needs to exist for the life of a domain, you may want to use a password generator.
+
+Example:
+
+  ```
+  kubectl -n MY-DOMAIN-NAMESPACE \
+    create secret generic MY-DOMAINUID-runtime-encrypt-secret \
+    --from-literal=password=welcome1
+  kubectl -n MY-DOMAIN-NAMESPACE \
+    label secret MY-DOMAINUID-runtime-encrypt-secret \
+    weblogic.domainUID=MY-DOMAINUID
+  ```
+
+Corresponding domain resource snippet:
+
+  ```
+  configuration:
+    model:
+      runtimeEncryptionSecret: MY-DOMAINUID-runtime-encrypt-secret
+  ```
+
+#### 5. Secrets for model macros
+
+Create additional secrets as needed by macros in your model files. For example, these can store database URLs and credentials that are accessed using `@@SECRET` macros in your model that reference the secrets.  For a discussion of model macros, see [Model files]({{< relref "/userguide/managing-domains/model-in-image/model-files.md" >}}).
+
+#### 6. Domain resource attributes
+
+The following domain resource attributes are specific to Model in Image domains.
+
+| Domain Resource Attribute                    |  Notes |
+| -------------------------                    |  ------------------ |
+| `domainHomeSourceType`                       |  Required. Set to `FromModel`. |
+| `domainHome`                                 |  Must reference an empty or non-existent directory within your image. Do not include the mount path of any persistent volume. Note that Model in Image recreates the domain home for a WebLogic pod every time the pod restarts.|
+| `configuration.model.wdtConfigMap`           | Optional. Set if you have stored additional models in a config map as per [Optional WDT model config map](#3-optional-wdt-model-config-map). |
+| `configuration.model.secrets`                | Optional. Set this array if your image or config map models contain macros that reference custom Kubernetes secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
```

Comment:
There is no "configuration.model.secrets"; model does not have "secrets", but configuration does.

Reply:
Fixed.